### DISCUSSION TITLE
Transferring Borg calendar to another computer duplicates ToDo entries

### DISCUSSION THREAD

**Discussion by DougBain** [mikeberger/borg_calendar#189] [Q&A]
To keep my travelling laptop and desktop PC in sync I transfer Borg calendar from the PC to my laptop with Import/Export=> Export XML to USB and then Import Entire Backup Zipfile from USB stick to the laptop. However I then find the ToDo entries duplicated on the laptop.
I reverse the procedure on return from a journey.
After several journeys there are multiple replicates of the ToDo list on PC and laptop which are a pain to remove.
Is there a proper way to avoid duplicating during transfer or a simple way of deleting all replicates except one?

**Comment by mikeberger** [comment]
I keep my database on google drive and don't even think about this kind of issue. I've also used One Drive and Dropbox in the past (dropbox for ease on linux). I just have to make sure that my laptop syncs before using another computer if it has been used without an internet connection while I made changes in Borg. 

If you are worried about data privacy and don't trust the file sharing service, then you would want to delete your existing database before importing. Import has no duplicate detection. So, I'd find the existing database folder and rename it (without borg running), then do the import, then delete or archive the old folder.
